packages/playwright-test/src/loader.ts

```diff
@@ -142,8 +124,6 @@ export class Loader {
       (config as any).screenshotsDir = path.resolve(configDir, (config as any).screenshotsDir);
     if (config.snapshotDir !== undefined)
       config.snapshotDir = path.resolve(configDir, config.snapshotDir);
-    if (config.webServer)
```

Comment:
Why removing this?

Reply:
`this._options.cwd = this._options.cwd ? path.resolve(configDir, this._options.cwd) : configDir;` is also in the plugin setup. Looks redundant.